app/views/menus/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-4 ">
+          <div class="card-deck mb-3 text-center">
+            <div class="card mb-4 box-shadow" id="js_main">
+              <div class="card-header">
+                <h4 class="my-0 font-weight-normal">Menu creator</h4>
+              </div>
+              <%= simple_form_for @facade.menu, html: { id: "myForm" } do |f| %>
```

Comment:
be conventional. Go with `camelCase` id/classes namings or `kebab-case`. But not `snake_case`